src/v2/Components/FlashBanner/index.tsx

```diff
@@ -84,6 +83,7 @@ const TrackedFlashBanner = track({
 
 export const FlashBannerQueryRenderer: React.FC = () => {
   const { relayEnvironment, user } = useSystemContext()
+  if (isServer) return null
```

Comment:
👍

Reply:
Is the `isServer` check needed for `SystemQueryRenderer`? (i thought that was one of the main reasons to use it, it will make sure it is only client-side for you).

Reply:
Just getting caught up here – @erikdstock – maybe I'm missing in the GH comment thread, but what is the goal of this `isServer` LOC? To be explicit that this will not work in a SSR context?

Reply:
Because below i essentially do `return user ? <SystemQueryRenderer /> (which blocks rendering until on client) `: <FlashBanner />` (which renders immediately and contains a reference to `window`, we need to make sure it will never render on client in the second case, so better to just be explicit about it all up front.